executable-spec/src/Cardano/Ledger/Spec/STS/Update/Data.hs

```diff
@@ -131,10 +159,26 @@ calcCommit sip =
       --`mappend` (myshow . sipHash $ sip)
       `mappend` (myshow . Core.hash $ sip)
 
+-- | Ideation phase environment
+data EnvIdeation
```

Comment:
About naming of environment and states, I'd rather use `State` or `St` everywhere, and then use them qualified, since that brings more uniformity, and avoids the suffix within the module.